build.gradle

```diff
@@ -170,33 +170,31 @@ ext {
 
     // Dependency versions
 
-    awaitilityVersion = '4.0.3'
+    awaitilityVersion = '4.1.0'
```

Comment:
i am kinda curious why we dont use the platform depencies for CLI

Reply:
Because the CLI is an Open-Source project, we can't include the private platform project, because no external contributor could build the project in this case. 

We might want to publish the platform in the future, but for now the cli keeps it own dependencies.